install_config/cluster_metrics.adoc

```diff
@@ -720,6 +720,32 @@ Information about scaling cluster metrics capabilities is available in the
 xref:../scaling_performance/scaling_cluster_metrics.adoc#cluster-metrics-scaling-openshift-metrics-pods[Scaling and
 Performance Guide].
 
+[[metrics-logging]]
+== Integration with Aggregated Logging
+
+Hawkular Alerts might be connected to the Aggregated Logging's Elasticsearch to react on log events. By default,
```

Comment:
> what is meant by "react on log events"?

Elasticsearch creates log entries (log events), which are later processed by Hawkular Alerts. As outcome of this processing, Hawkular Alerts might trigger a new alert if it detects that it's an `ERROR`.